cms/envs/common.py

```diff
@@ -1089,6 +1089,10 @@
         'component': 'openassessment',
         'boilerplate_name': None,
     },
+    {
```

Comment:
Please also add unit test coverage for this "masquerading". Search for  test_xblock_masquerading_as_problem in CMS.